plugins/module_utils/ansible_freeipa_module.py

```diff
@@ -225,7 +225,7 @@ def date_format(value):
     raise ValueError("Invalid date '%s'" % value)
 
 
-def compare_args_ipa(module, args, ipa):  # noqa
+def compare_args_ipa(module, args, ipa, ignore=[]):  # noqa
```

Comment:
The use of mutable default values may cause issues later. Use `ignore=None` and it the implementation use `if ignore is None: ignore = []`.

Even if we do not modify ignore in `compare_args_ipa`, we don't accidentally get a non-empty list after multiple calls, if we later start to modify it.

Reply:
Hey, thanks for the peer review.  I have implemented your suggestions.